nservicebus/licensing/index.md

```diff
@@ -32,40 +32,41 @@ partial: validity
 partial: limitations
 
 
-## License Management
+## License management
+
+There are several options available for installing the license file.
 
-partial: code
 
+partial: code
 
-### Using the Registry
 
-NOTE: Using the [NServiceBus PowerShell Module](/nservicebus/operations/management-using-powershell.md) is the preferred and simplest method of adding the license file. 
+### Using the registry
 
-Using the registry allows all platform tools (endpoints, [ServiceControl](/servicecontrol/license.md), and [ServiceInsight](/serviceinsight/license.md)) to access the same license.
+Storing the license file in the registry allows all platform tools (endpoints, [ServiceControl](/servicecontrol/license.md), and [ServiceInsight](/serviceinsight/license.md)) to access the same license.
 
 partial: registry
 
 partial: registry-caveats
 
 
-### Using a License sub-directory
+### Using a license subdirectory
 
 A file located at `[AppDomain.CurrentDomain.BaseDirectory]/License/License.xml` will be automatically detected.
```

Comment:
btw what happens for SC/SI then? might be good to also mention/link to more info here